exercises/java-collections/src/main/java/com/cbfacademy/CollectionsExercises.java

```diff
@@ -46,13 +121,45 @@ public HashMap<Integer, String> useHashMap() {
         //  - add {5, JavaScript} entry to the map
         //  - add {6, Rust} entry to the map
         //  - determine the set of keys from the map and print it on the screen
-        //  - determine the set of keys from the map and print it on the screen
+        //  - determine the set of value from the map and print it on the screen
         //  - determine whether the map contains "English" as a language and print the result on the screen
         //  - return the map
-        throw new RuntimeException("Not implemented");
+
+
+
+        HashMap<Integer, String> hashMap = new HashMap<>();
+        hashMap.put(1, "TypeScript");
+        hashMap.put(2, "Kotlin");
+        hashMap.put(3, "Python");
+        hashMap.put(4, "Java");
+        hashMap.put(5, "JavaScript");
+        hashMap.put(6, "Rust");
+
+        Set<Integer> keys = hashMap.keySet(); //keySet() method to obtain a Set<Integer> of all the keys in the map and print it.
+        System.out.print(keys);
+
+        /*values() method to obtain a Collection<String> of all the values in the map, 
+        and then convert it to a HashSet<String> to remove duplicate values.
+        A LinkedHashSet preserves the order of elements based on their insertion order, 
+        just like a LinkedHashMap. This ensures that the order of the values in the 
+        output will match the order they were inserted into the map. */
+
+        Set<String> values = new LinkedHashSet<>(hashMap.values()); 
+        System.out.print(values);
+
+        String languageToCheck = "English";
+        boolean containsLanguage = values.contains(languageToCheck);
+        System.out.print(containsLanguage);
```

Comment:
HashMap has a `containValues()` method. How could we use it here, so that these three lines could become one line of code instead?